### ISSUE TITLE
NIOTSChannelOptions doesn't exist

### ISSUE THREAD

**Issue by ldewailly** [apple/swift-nio-transport-services#4]
### Expected behavior
The documentation references `NIOTSChannelOptions` to access TPC or TLS configuration

### Actual behavior
The above class does not exist

### Steps to reproduce
* https://github.com/apple/swift-nio-transport-services/blob/master/Sources/NIOTransportServices/NIOTSConnectionBootstrap.swift#L81
* https://github.com/apple/swift-nio-transport-services/blob/master/Sources/NIOTransportServices/NIOTSConnectionBootstrap.swift#L91
* https://github.com/apple/swift-nio-transport-services/blob/master/Sources/NIOTransportServices/NIOTSListenerBootstrap.swift#L124
* https://github.com/apple/swift-nio-transport-services/blob/master/Sources/NIOTransportServices/NIOTSListenerBootstrap.swift#L134

### If possible, minimal yet complete reproducer code (or URL to code)

n/a

### SwiftNIO version/commit hash

897eb363f5ffc6ca995d57182cb1bf9a8f43a867 (release 0.2.0)

### Swift & OS version (output of `swift --version && uname -a`)

n/a

**Comment by Lukasa** [comment]
Yeah, those docs are wrong and should be removed: accessing the options once the channel is created is not supported.
